### ISSUE TITLE
Icons for Growl and HardwareGrowler 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tazounet/growl#633]
```
What steps will reproduce the problem?
1. Install 10.10
2. Enable Dark Theme

What is the expected output?
Growl and Hardware Growler menubar icons distinguishable from menu bar

What do you see instead?
Icons blend into the menu bar and are barely visible.

What version of the product are you using? On what operating system?
Growl-2.1.3, Hardware Growler-2.2, OS X 10.10 (14A389)
```

Original issue reported on code.google.com by `ramer...@gmail.com` on 17 Oct 2014 at 6:45

**Comment by adisor19** [comment]
Can confirm hat I have the same issue. Hardware Growler's icon is not compatible with dark theme on Yosemite.

**Comment by tazounet** [comment]
Corrected in commit 7adef1c8c64de007472ac4213954e8e23883ad43
